main.nf

```diff
@@ -134,8 +134,8 @@ if (params.pico){
 }
 
 // Validate inputs
-if (params.aligner != 'star' && params.aligner != 'hisat2'){
-    exit 1, "Invalid aligner option: ${params.aligner}. Valid options: 'star', 'hisat2'"
+if (params.aligner != 'star' && params.aligner != 'hisat2' && params.aligner != 'salmon'){
```

Comment:
I think we should allow to use genomic and transcriptomic alignment at the same time. The QC for genomic aligner gives a lot more information. I would just add salmon as another quantification, as an extra option...For me it would be very valuable to run both strategies.

Reply:
Personally, I think that should probably go in another PR @lpantano . That will require some more testing. Especially to make sure that you arent overwriting channels between the two alignment methods for the downstream processes.

Reply:
wait, right now is doing hisat2 or star and salmon right? after @olgabot changes?  or now the current DEV branch will do salmon or hisat2 or star? I thought salmon was an alternative if transcriptome was available but not interfering with hisat2 or star

Reply:
I think thats what needs to be tested properly 👍

Reply:
but I don't know why "**Salmon**" is an option for `params.aligner` now, what happens if you set that to "Salmon" and there is no `transcriptome`? and if you set up that to "**hisat2**" and you have `params.transcriptome`, **Salmon** will run anyway beside the other aligner.

Reply:
Also, need to add `--aligner` to usage at top of `main.nf`

Reply:
"not found" is probably better for this line:
https://github.com/nf-core/rnaseq/blob/87185bac003497b4e4fa6a0bedc0ce170d24db42/main.nf#L176